### DISCUSSION TITLE
Question: get info/parameters from Cozmo (camera for example) ?

### DISCUSSION THREAD

**Discussion by snowgoon88** [zayfod/pycozmo#42] [Q&A] [answered]
Hello everyone,

is it possible to get some parameters from the robot ?
For example, I have read in the doc that one can set the aperture, gain and autogain for the camera using `protocol_encoder.SetCameraParams`. But is there a way to read this parameters from Cozmo ? A `GetCameraParams` ?

**Comment by zayfod** [comment, marked as answer]
Good question. I wonder about the same myself.

At present, I am unaware of a message that allows retrieving the state of autoexposure or current exposure and gain settings.

Such a message may very well exist.
